### PR TITLE
th-263: Log out implementation

### PR DESCRIPTION
**Pull request by h0wter**
Added the ability to log out a user with token deletion from local storage and database.
Changed the `li` component in the burger menu list, when a picture was used the click on the element triggered only on a small area of the picture, now the click triggers on the entire area of the element.

### CODE REVIEW

**Comment by h0wter** [comment]
Removed the redirect as some pages are accessible to unregistered users.

File: frontend/src/packages/auth/auth-api.ts [hunk 1]
```diff
@@ -77,6 +77,14 @@ class AuthApi extends HttpApi {
       CustomerSignUpResponseDto | BusinessSignUpResponseDto
     >();
   }
+
+  public async logOut(): Promise<unknown> {
+    return await this.load(this.getFullEndpoint(AuthApiPath.LOGOUT, {}), {
+      method: 'GET',
+      contentType: ContentType.JSON,
+      hasAuth: true,
```
Comment (albsbz) [inline frontend/src/packages/auth/auth-api.ts:85]: will it work if token expired?

Comment (h0wter) [reply frontend/src/packages/auth/auth-api.ts:85]: I have connected a JWT validation strategy on endpoint, and it checks for token relevance. With an expired one, the user will not be able to logout.

**Comment by h0wter** [comment]
Fixed the background for the password hide/display icon to be transparent during autocomplete.

**Review by kalinkaaaa14 — APPROVED** [review]

**Review by Druxight — APPROVED** [review]

**Pull request merged**